src/content/blog/astro-4120.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: "Astro 4.12: Server Islands"
+description: Astro 4.12 is now available! This release includes a first look at Server Islands, improvements to pagination and syntax highlighting, and more.
+publishDate: "July 18, 2024"
+authors:
+  - erika
+  - matthew
+coverImage: "/src/content/blog/_images/astro-4120/header_astro_4_12.webp"
+socialImage: "/src/content/blog/_images/astro-4120/og_image_4_12.webp"
+lang: "en"
+---
+
+import BlogContentImage from "/src/components/BlogContentImage.astro";
+import BlogSummary from "/src/components/BlogSummary.astro";
+import islandDiag from "/src/content/blog/_images/future-of-astro-server-islands/dark-mode-server-islands-diag.webp";
+
+<BlogSummary>
+  Astro 4.12 is out now! This release includes a first look at [Server Islands](https://astro.build/blog/future-of-astro-server-islands/), as well as improvements to pagination and syntax highlighting, and more.
```

Comment:
less clear that "first look" means "you can try it today". I'd say "experimental release" instead (or something similar)

Also, add a quick 1-sentence explanation of what Server Islands are. The goal should be that if someone only reads this sentence, they take away the most important bits.

> Astro 4.12 is out now! This release includes our first experimental release of [Server Islands](https://astro.build/blog/future-of-astro-server-islands/) $WITH_TLDR. Improvements to pagination and syntax highlighting are also included.